### PR TITLE
Validate Ethereum addresses in input fields

### PR DESCRIPTION
**Pull request by skddc**
Adds address validation to the address inputs.

Importing the function from `web3-utils`, because `ethers.js` doesn't have any such utility function. :slightly_frowning_face:

### CODE REVIEW

**Comment by bumi** [comment]
ethers has that getchcksumaddress function that does some check: https://github.com/ethers-io/ethers.js/blob/061b0eae1d4c570aedd9bee1971afa43fcdae1a6/src.ts/utils/address.ts#L22

I guess that duplication of `isValidAccount` can not be prevented as we need that method in both files?

File: app/components/add-contributor/component.js [hunk 1]
```diff
@@ -1,22 +1,27 @@
...
 export default Component.extend({
 
   kredits: service(),
 
   attributes: null,
 
-  // TODO: add proper address validation
-  isValidAccount: notEmpty('account'),
+  isValidAccount: computed('ethAddress', function() {
+    return isAddress(this.ethAddress);
```
Comment (bumi) [inline app/components/add-contributor/component.js]: why is this now `ethAddress` and was `account` before?

**Comment by skddc** [comment]
> ethers has that getchcksumaddress function that does some check: https://github.com/ethers-io/ethers.js/blob/061b0eae1d4c570aedd9bee1971afa43fcdae1a6/src.ts/utils/address.ts#L22

That's not a check, that's actually creating a checksum address from a non-checksum one.

File: app/components/add-contributor/component.js (hunk 1 above)
Comment (skddc) [reply app/components/add-contributor/component.js]: Typo. Thanks.

**Comment by skddc** [comment]
> I guess that duplication of `isValidAccount` can not be prevented as we need that method in both files?

It can, but that would be way more complex and require more code than the de-duplication is worth. In essence, the `isAddress` is what's de-duplicated here, and using it on this or that property is just a declarative action.

**Comment by skddc** [comment]
Good to merge.

**Pull request merged**